test/e2e/scenario/logs.scenario.ts

```diff
@@ -74,22 +74,15 @@ describe('logs', () => {
       })
     })
 
-  createTest('read only the first bytes of the response')
+  createTest('keep only the first bytes of the response')
     .withLogs({ forwardErrorsToLogs: true })
     .run(async ({ serverEvents, baseUrl, servers }) => {
       await browserExecuteAsync((done) => {
-        fetch('/throw-large-response').then(
-          (response) => {
-            // The body stream needs to be cancelled, else the browser will still download the whole
-            // response even if it is unused.
-            response
-              .body!.getReader()
-              .cancel()
-              .catch((error) => console.log(error))
+        fetch('/throw-large-response')
+          .then((response: Response) => response.blob())
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏adding a comment to explain what is your intent with `response.blob()` could be helpful.